setup.py

```diff
@@ -440,6 +470,9 @@ def readme():
     license=LICENSE,
     # Ensure the packages are being found in the correct locations
     package_dir={"sisl_toolbox": "toolbox"},
+    package_data={
+        "sisl.viz.plotly.gui": ["build/*"],
```

Comment:
I am not sure I understand this?

When one issues a sisl build the build-procedure will create a build folder. This folder contains basically the entire build of sisl.
It seems to me that having `build/*` as package-data is really bad... Why is this needed?

Reply:
I think this was something that you included, because I don't know how it works. You did it so that the build files of the GUI were included in the install and you could use it (otherwise you could only use the GUI by doing `pip install -e`).

The main reason we had to incorporate the GUI's build in the repo was that, in order to build it one needs [nodejs](https://nodejs.org/es/), and this is probably not a good thing to have as a requirement for a python package.

However, I realised that TravisCI can build it. Currently I have the sources in a [separate repo](https://github.com/pfebrer96/sislGUIpublic). I already use travis CI to build the GUI and deploy it to https://sisl-siesta.xyz. What do you think it could be the best way to do it?

Reply:
Thanks for reminding me!! Then I should play more with this.
Especially this puts some constraints on the release process, probably I should look into matplotlib's procedure.

I think it would be best to have the repo *clean* and then dev's should follow one requirement list, and users would get the full thing without problems. So we need some way of dealing with this. :)